packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -115,7 +117,7 @@ export function DatePicker({
   const datePickerClassNames = classnames(styles.datePicker, {
     [styles.inline]: inline,
   });
-  const { pickerRef } = useEscapeKeyToCloseDatePicker(open, ref);
+  const { pickerRef } = useEscapeKeyToCloseDatePicker(open);
```

Comment:
the `ref` removed from here wasn't needed for ESC key functionality; ESC key handler only needs `pickerRef` to call `setOpen(false)` and doesn't need the wrapper's `ref`.